chef_master/source/platforms.rst

```diff
@@ -172,9 +116,6 @@ The following table lists the Foundational platforms for the Chef DK:
    * - Platform
      - Architecture
      - Version
-   * - Debian
-     -
-     - ``7.x``, ``8.x``
```

Comment:
Does this mean we can stop building chefdk on debian? :D

Reply:
Maybe? Do we know how this even started? Like what customer needed it? (you can message me privately if you have that info, since this is a public PR)